Sources/Client.swift

```diff
@@ -1,10 +1,13 @@
+import MirrorCoreSDK.EMSConfig
+
 @objc
 public class MirrorClient: NSObject, ConnectivityObserver {
 
     @objc
-    public override init() {
+    public init(appID: String = "", appToken: String = "") {
```

Comment:
In this current form, it's possible to call the constructor:

- with no params
- with 1 param (non-empty app ID, empty app token)
- with 2 params (non-empty app ID and token)

The middle one doesn't make much sense. What we want is:

- no params (if you're using Mirror in the Development mode, and thus Cloud authentication is optional)
- both app ID and token (if you wanna use Cloud features or have encryption/security enabled on Mirror, and thus Cloud authentication is required)

So maybe instead of default params, just overload the constructor, one param-less version, and one that requires two params?

Reply:
I implemented a `convenience init?()` that returns `nil` if given empty strings.